__tests__/Unit/hooks/tasksApi.test.tsx

```diff
@@ -214,4 +214,37 @@ describe('useUpdateTaskMutation()', () => {
             message: 'Task not found',
         });
     });
+
+    test('updates a task with the isDevEnabled set to true', async () => {
+        const taskId = '1eJhUW19D556AhPEpdPr';
+        const { result, waitForNextUpdate } = renderHook(
```

Comment:
please also test what will happen if isDevEnabled is set to false

Reply:
Sure